Earthfile

```diff
@@ -2,56 +2,7 @@ VERSION 0.7
 
 INSTALL_DIND:
```

Comment:
@vladaionescu 
I'm not even sure we need to keep this for backwards compatibility since we're going to cut a new release/tag for `utils` and discontinue the global tag (which users can still use).
Thoughts?

Reply:
I would keep it, because we have many users of INSTALL_DIND who don't use a tag at all. (We should update our docs too to suggest using a tag)

I wonder if we should add a `RUN echo <some-warning-message>` when using the outter INSTALL_DIND, to invite the users to use the new version directly + a tag.

Reply:
I think that'd a good idea